includes/query_processor.inc

```diff
@@ -339,6 +339,24 @@ class IslandoraSolrQueryProcessor {
       $datastream_field = variable_get('islandora_solr_datastream_id_field', 'fedora_datastreams_ms');
       $object_label = variable_get('islandora_solr_object_label_field', 'fgs_label_s');
       if (!empty($object_results)) {
+        if (variable_get('islandora_solr_search_navigation', FALSE)) {
+          $id = substr(str_shuffle("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"), 0, 10);
```

Comment:
https://api.drupal.org/api/drupal/includes!bootstrap.inc/function/drupal_random_bytes/7 is more normal, can we not use it?